### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by meyerweb**
It’s time to merge!

----

Site preview: https://igalia.github.io/wpewebkit.org/redesign/

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		grid-template-columns: 1fr 1fr 1fr;
+	}
+}
+</style>
+
+<nav class="sidebar">
+<ul>
+<li><a href="{{ '/about/supported-hardware.html' | url }}">Supported Hardware</a></li>
+<li><a href="{{ '/about/a-good-choice.html' | url }}">Why Choose WPE?</a></li>
+<li><a href="">WPE in Action</a></li>
```
Comment (bkardell) [inline about/index.md:29]: should this go to ... #wpe-in-action?

File: about/supported-hardware.md [hunk 2]
```diff
@@ -1,19 +1,121 @@
...
+	border-top: 1px solid silver;
+}
+</style>
 
-[[toc]]
+<nav class="sidebar">
+<ul>
+<li class="currentPage"><a href="{{ '/about/supported-hardware.html' | url }}">Supported Hardware</a></li>
+<li><a href="{{ '/about/a-good-choice.html' | url }}">Why Choose WPE?</a></li>
+<li><a href="{{ '' | url }}">WPE in Action</a></li>
```
Comment (bkardell) [inline about/supported-hardware.md:101]: Should this go to `/about/index.html#wpe-in-action"`?

File: about/index.md (hunk 1 above)
Comment (meyerweb) [reply about/index.md]: Yep, fixed.

**Review by bkardell — APPROVED** [review]
My individual comments are already addressed, so lgtm

**Pull request merged**